src/mbgl/renderer/render_layer.hpp

```diff
@@ -16,16 +16,24 @@ class PropertyEvaluationParameters;
 class UploadParameters;
 class PaintParameters;
 class RenderSource;
-class RenderLayerSymbolInterface;
 class RenderTile;
 class TransformState;
+class GeometryTile;
 
 class LayerRenderData {
 public:
     std::shared_ptr<Bucket> bucket;
     Immutable<style::LayerProperties> layerProperties;
 };
 
+class LayerPlacementData {
+public:
+    std::reference_wrapper<Bucket> bucket;
+    std::reference_wrapper<RenderTile> tile;
+    bool pitchWithMap;
```

Comment:
Members could be `const`.

Reply:
no need, it's anyway passed as const ref